command_execution.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import "os/exec"
+
+func f(string x) {
```

Comment:
__[staging][gometalinter]__
* [deadcode] f is unused
* [gotype] undeclared name: x
* [interfacer] undeclared name: x
* [varcheck] unused global variable undeclared name: x
* [errcheck] error return value not checked (undeclared name: x)
* [unconvert] undeclared name: x
* [aligncheck] undeclared name: x
* [structcheck] unused struct field undeclared name: x

<!-- Comment by [SideCI](https://sideci.com) -->